app/models/profile.rb

```diff
@@ -184,7 +185,7 @@ def fraud_deactivation_reason?
   end
 
   def in_person_verification_pending?
-    in_person_verification_pending_at.present?
+    in_person_verification_pending_at.present? && !in_person_enrollment&.expired?
```

Comment:
Why is InPersonEnrollment not Expired checked rather than InPersonEnrollment is pending?

Reply:
That's a good call, I'll update to have them in sync.